test/routeBuilderSpec.js

```diff
@@ -237,6 +244,13 @@ describe('RouteBuilder', function () {
             })
           })
         }
+
+        // Populate the curve cache.
```

Comment:
I think this is not needed here, nor on [line 229/236 above](https://github.com/interledgerjs/ilp-connector/pull/355/files#diff-9b51c24e991fec48cf7254e7394a6843R236) since the route [already has a curve](https://github.com/interledgerjs/ilp-connector/pull/355/files#diff-9b51c24e991fec48cf7254e7394a6843R233)?